v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/v2g_liberty.py

```diff
@@ -786,7 +791,7 @@ async def set_records_in_chart(self, chart_line_name: ChartLine, records: dict):
     ######################################################################
 
     async def __pong(self, event, data, kwargs):
-        self.hass.fire_event("ping.result")
+        await self.hass.fire_event("ping.result")
```

Comment:
Since this is "fire and forget", there is no real need to await this call.

Reply:
PyCharm (that i still use every now and then) gave a warning it should be awaited. But, I think you are right, I'll remove.

Reply:
It does not really hurt to include it. So if that makes PyCharm happy ...